File: Codigo/Obligatorio/codigo/ArenaGestor/ArenaGestor.APIContracts/Band/BandInsertBandDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace ArenaGestor.APIContracts.Band
{
    public class BandInsertBandDto
    {
        public string Name { get; set; }

        public DateTime StartDate { get; set; }

        public int GenderId { get; set; }

        public List<BandInsertArtistDto> Artists { get; set; }
    }
}

File: Codigo/Obligatorio/codigo/ArenaGestorFront/src/app/models/Bands/BandInsertBandDto.ts
import { BandInsertArtistDto } from "./BandInsertArtistDto";

export class BandInsertBandDto {
    name: String = "";
    startDate: Date = new Date();
    genderId: Number = 0;
    artists: Array<BandInsertArtistDto> = new Array();
}